pkg/core/blockchain.go

```diff
@@ -504,6 +504,15 @@ func (bc *Blockchain) persist() error {
 			"blockHeight":     bHeight,
 			"took":            time.Since(start),
 		}).Info("blockchain persist completed")
+
+		BCMetrics := &BlockChainMetrics{}
+		BCMetrics.Update(
+			bHeight,
+			bc.persistedHeight,
+			storedHeaderHeight,
+			bc.memPool.capacity,
+			len(bc.memPool.unsortedTxn),
+			len(bc.memPool.unverifiedTxn))
```

Comment:
Same thing with `memPool`, this is not the place where it's updated.

Reply:
made updated inside mempool on change operations